src/app/components/wrapper-components/part-creation/part-creation.component.spec.ts

```diff
@@ -167,6 +167,7 @@ describe('PartCreationComponent:', () => {
                 joueursDAOMock.update('opponent', { state: 'offline' });
                 testUtils.detectChanges();
                 tick();
+                flush();
 
```

Comment:
wa... what happend there ?

Reply:
mise à jour ng 11 qui a cassé des tests (timer is still present), qui nécéssitent ici soit un tick(1000) soit un flush() ! Sacrés tests async...

Reply:
tick me semble plus propre, si ça marche!
J'espère que ces changements vont vers le bon sens et pas vers le wtf (ou d'un wtf à l'autre) !